lib/Runtime/Library/JsBuiltIn/JsBuiltIn.js

```diff
@@ -13,14 +13,6 @@
     var Symbol = platform.Symbol;
     var CreateObject = platform.builtInJavascriptObjectEntryCreate;
 
-    let FunctionsEnum = {
-        ArrayValues: setPrototype({ className: "Array", methodName: "values", argumentsCount: 0, forceInline: true /*optional*/, alias: "Symbol.iterator" }, null),
-        ArrayKeys: setPrototype({ className: "Array", methodName: "keys", argumentsCount: 0, forceInline: true /*optional*/ }, null),
-        ArrayEntries: setPrototype({ className: "Array", methodName: "entries", argumentsCount: 0, forceInline: true /*optional*/ }, null),
-        ArrayIndexOf: setPrototype({ className: "Array", methodName: "indexOf", argumentsCount: 1, forceInline: true /*optional*/ }, null),
-        ArrayFilter: setPrototype({ className: "Array", methodName: "filter", argumentsCount: 1, forceInline: true /*optional*/ }, null),
-    };
-
```

Comment:
Drive by clean up - this FunctionsEnum object appears to be left over from a previous way this file was structured - it's no longer used - the information it was providing is all now given as parameters for `registerChakraLibraryFunction`.